unblob/pool.py

```diff
@@ -85,18 +115,31 @@ def close(self):
     def _clear_input_queue(self):
         try:
             while True:
+                logger.warning("getnowait")
                 self._input.get_nowait()
         except queue.Empty:
             pass
 
     def _request_workers_to_quit(self):
         for _ in self._procs:
+            logger.warning("sentinel")
             self._input.put(_SENTINEL)
 
     def _wait_for_workers_to_quit(self):
         for p in self._procs:
+            logger.warning("join")
```

Comment:
Is this really a warning case? It wouid be printed `-p` times even for normal shutdown